modules/libdeflate/1.18/patches/add_build_file.patch

```diff
@@ -0,0 +1,59 @@
+--- /dev/null
++++ b/modules/libdeflate/1.18/patches/BUILD.bazel
+@@ -0,0 +1,56 @@
++# SPDX-License-Identifier: BSD-3-Clause
++# Copyright (c) Contributors to the OpenEXR Project.
++
++licenses(["notice"])
++
++exports_files(
++    glob(["lib/**"]) + ["common_defs.h"],
```

Comment:
Is this necessary? It makes the total list of files part of the API of this module.

Reply:
Seems not to be necessary - was some old clutter - removednow